src/current/v23.1/migration-overview.md

```diff
@@ -5,87 +5,304 @@ toc: true
-### Schema design best practices
+### Cutover strategy
 
-Consider the following CockroachDB attributes and best practices:
+*Cutover* is the process of switching application traffic from the legacy database to CockroachDB. Consider the following:
 
-- We recommend that you always have an explicit primary key defined on every table. For more information, see [Primary key best practices](schema-design-table.html#primary-key-best-practices).
+- Will you perform the cutover all at once, or incrementally (e.g., by a subset of users, workload, or tables)?
+
+	- Switching all at once generally follows a [downtime window](#downtime-window). Once the data is migrated to CockroachDB, you "flip the switch" to route application traffic to the new database, thus ending downtime.
+
+	- Migrations with [zero or near-zero downtime](#zero-or-near-zero-downtime) can switch either all at once or incrementally, since writes are being synchronously replicated and the system can be gradually migrated as you [validate the queries](#validate-queries).
+
+- Will you have a fallback plan in which you reverse ("roll back") the migration from CockroachDB to the legacy database? A fallback plan enables you to fix any issues or inconsistencies that you encounter during or after cutover, then retry the migration.
+
+#### All at once (no rollback)
+
+This is the simplest cutover plan, since you won't need to develop and execute a fallback method. It is typically used with [lift-and-shift migrations](#lift-and-shift).
+
+After moving all of the data from the legacy database to CockroachDB, you switch application traffic to CockroachDB. Since you cannot roll back the migration after switching over, you **must** [validate your queries](#validate-queries) first.
+
+#### All at once (rollback)
+
+This method can be used for [lift-and-shift migrations](#lift-and-shift), as well as for [zero-downtime migrations](#zero-downtime).
+
+In addition to moving data to CockroachDB, data is also replicated from CockroachDB back to the legacy database. Continuous replication is already possible when performing a zero-downtime migration that dual writes to both databases. Otherwise, you will need to ensure that writes are consistent in both directions.
+
+#### Phased rollout
+
+Also known as the ["strangler fig"](https://en.wikipedia.org/wiki/Strangler_fig) approach, a phased rollout migrates a portion of your users, workload, or tables over time. Meanwhile, the application continues writing to the legacy database.
+
+This approach enables you to take your time with the migration, and to pause or roll back as you [monitor the migration](#set-up-monitoring-and-alerting) for issues and performance. A phased rollout has the least risk and user impact, since you can control the blast radius of your migration by routing traffic for a subset of users or services. However, it has the highest development effort, since you will need to figure out how to migrate in phases while maintaining consistency.
```

Comment:
"least risk" is debatable. Its the least risk to the business but the most technical risk since its complicated. I'd say something like "reduced business risk and user impact at the cost of increased implementation risk."